docs/how-to/using-omnitrace-api.rst

```diff
@@ -0,0 +1,294 @@
+.. meta::
+   :description: Omnitrace documentation and reference
+   :keywords: Omnitrace, ROCm, profiler, tracking, visualization, tool, Instinct, accelerator, AMD
+
+****************************************************
+Using the Omnitrace API
+****************************************************
+
+The following example shows how a program can use the Omnitrace API to analyze a program.
```

Comment:
@dgaliffiAMD Could you review the changes I made and see if this makes more sense for the user. I'm more than happy to make additional changes if required. Thanks!